style.css

```diff
@@ -346,3 +345,89 @@ summary:hover .summary-up svg {
   font-size: 15px;
   font-weight: 500;
 }
+
+/* contact-me section */
+
+.contact-background {
+  position: absolute;
+  width: 100%;
+  height: 779px;
+  background-color: #6070ff;
+  background-image: url("Images/contact\ form\ shapes-mobile@2x.png");
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: 0 50px;
+  border-radius: 90px 0 0 0;
+  z-index: -999;
+}
```

Comment:
**[REQUIRED CHANGES]**

- Please, let's use the right background image for our contact form to match the figma design
- Please, let's also add the missing black line to match the figma design